dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -12,6 +12,18 @@ def load_user
     @user = current_user
   end
 
+  # GET /api/v1/users/<user_id>/school_name
+  def get_school_name
+    render json: {
+      school_name: current_user&.school
+    }
+  end
+
+  # PATCH /api/vi/users/<user_id>/update_user_info
+  def update_user_info
+    current_user&.last_seen_school_info_interstitial = DateTime.now
```

Comment:
make sure to save after updating this column on the user! calling `current_user&.save` after this line should do